pkgs/tools/networking/qrcp/default.nix

```diff
@@ -5,19 +5,27 @@
 
 buildGoModule rec {
   pname = "qrcp";
-  version = "0.7.0";
+  version = "0.8.1";
 
   src = fetchFromGitHub {
     owner = "claudiodangelis";
     repo = "qrcp";
     rev = version;
-    sha256 = "0rx0pzy7p3dklayr2lkmyfdc00x9v4pd5xnzydbjx12hncnkpw4l";
+    sha256 = "001w15hj5xb7p9gpvw1216lp26g5018qdi8mq6i84akb7zfd2q01";
   };
 
-  vendorSha256 = "0iffy43x3njcahrxl99a71v8p7im102nzv8iqbvd5c6m14rsckqa";
+  vendorSha256 = "1hn8c72fvih6ws1y2c4963pww3ld64m0yh3pmx62hwcy83bhb0v4";
 
   subPackages = [ "." ];
 
+  postInstall = ''
+    # generate completion scripts for qrcp
+    mkdir -p "$out/share/"{bash-completion/completions,fish/vendor_completions.d,zsh/site-functions}
+    $out/bin/qrcp completion bash > "$out/share/bash-completion/completions/qrcp"
+    $out/bin/qrcp completion fish > "$out/share/fish/vendor_completions.d/qrcp.fish"
+    $out/bin/qrcp completion zsh  > "$out/share/zsh/site-functions/_qrcp"
```

Comment:
```suggestion
    installShellCompletion --bash <($out/bin/qrcp completion bash)
    installShellCompletion --fish ($out/bin/qrcp completion fish)
    installShellCompletion --zsh ($out/bin/qrcp completion zsh)
```

Reply:
Resolve the conversation, if you fixed it